File: source/code/vbnet/idsite/jwt_for_oauth_req.vb
Dim jwt = "incoming_uri".Split("="c)(1)

Dim idSiteTokenExchangeRequest = OauthRequests _
    .NewIdSiteTokenAuthenticationRequest() _
    .SetJwt(jwt) _
    .Build()

Dim grantResponse = Await app.NewIdSiteTokenAuthenticator() _
    .AuthenticateAsync(idSiteTokenExchangeRequest)

' Token is stored in grantResponse.AccessTokenString

File: source/code/csharp/idsite/jwt_for_oauth_req.cs
var jwt = incoming_uri.Split('=')[1];

var idSiteTokenExchangeRequest = OauthRequests
    .NewIdSiteTokenAuthenticationRequest()
    .SetJwt(jwt)
    .Build();

var grantResponse = await app.NewIdSiteTokenAuthenticator()
    .AuthenticateAsync(idSiteTokenExchangeRequest);
